pkgs/servers/tt-rss/plugin-tumblr-gdpr/default.nix

```diff
@@ -2,19 +2,22 @@
 
 stdenv.mkDerivation rec {
   pname = "tt-rss-plugin-tumblr-gdpr";
-  version = "2.1";
+  version = "2.2";
 
   src = fetchFromGitHub {
     owner = "GregThib";
     repo = "ttrss-tumblr-gdpr";
     rev = "v${version}";
-    sha256 = "09cbghi5b6ww4i5677i39qc9rhpq70xmygp0d7x30239r3i23rpq";
+    sha256 = "sha256-oCdpgfOvXNui2KaS48WeyVy8Of6IlPOxQR6gmPiyH0s=";
   };
 
   installPhase = ''
-    mkdir -p $out/tumblr_gdpr
+    runHook preInstall
 
+    mkdir -p $out/tumblr_gdpr
     cp init.php $out/tumblr_gdpr
```

Comment:
Shouldn't this go into $out/share or something similar?

Reply:
No, that's not what the module expects.